kenware/Assessment
# #162711236Ft modify user endpoints

#### What does this PR do?
- Modify user endpoints

#### Description
- Modify user endpoints to add permission and authentications
- Add models and schemas
- add test
#### How should this be manually tested?
- Create a local copy `git fetch origin ft-user-endpoints-162711236`
- Checkout to the branch `git checkout ft-user-endpoints-162711236`
- Start the application with the command: `python manage.py runserver`
- On postman make a GET request to `{{base-url}}/api/v1/users/`
- An authenticated user should be able to carry out all crude operation on user endpoints
- To run tests use `python manage.py`
#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- [#162187176](https://www.pivotaltracker.com/story/show/162187176)
